grafana-datasource/compose.yaml

```
@@ -3,7 +3,7 @@ services:
     image: spiceai/spiceai
     container_name: spice
     ports:
-      - "5000:3000"
+      - "5000:8090"
```

```suggestion
      - "5000:5000"
```

This should be `5000` since the `--http` flag below is overriding the default to `5000`. You could remove the override and keep it as `5000:8090` or change it to `5000:5000`